client-common/src/tendermint/types/block_results.rs

```diff
@@ -53,24 +56,25 @@ pub struct Attribute {
 
 impl BlockResults {
     /// Returns transaction ids in block results
```

Comment:
This comment is outdated after the change

Reply:
ok, will update